test/unit/logic/block_reward.js

```diff
@@ -162,129 +174,133 @@ describe('BlockReward @slow', () => {
 		});
 
 		it('when height == 0 should return 0', () => {
-			return expect(blockReward.calcReward(0).equals(0));
+			return expect(blockReward.calcReward(0).isEqualTo(0));
```

Comment:
Similar here, should be `expect(something).to.be.true;`, there are many in this file.